api/components/flows/B2C/incremental_catalog_regular_push.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Retail 1.2.0 (Incremental Regular Push)
+description: The following is an illustrative flow to perform a transaction of a nature where a buyer would like to order customized item and it is delivered successfully and it is searched through incremental regular push
+reference: if any
+steps:
+  - summary: Search by city
+    api: search
+    description:  Search by city
+    reference: if any
+    example:
+      $ref: "../../Examples/B2C/search/search_by_city.yaml"
+    notes:
+      $ref: "../../notes/search/search_by_city.yaml"
+  - summary: Search by incremental regular push catalog refresh
+    api: search
+    description:  Search by incremental regular push catalog refresh
+    reference: if any
+    example:
+      $ref: "../../Examples/B2C/search/incremental_catalog_refresh_regular_push.yaml"
+  - summary: Stop incremental refresh
```

Comment:
on_search for incremental push will come first and then this example will come. As when buyer asks to stop sending incremental refresh, then seller does not send incremental refresh updates.